homeassistant/components/islamic_prayer_times/sensor.py

```diff
@@ -55,42 +56,22 @@ def icon(self):
     @property
     def state(self):
         """Return the state of the sensor."""
-        return self._state
-
-    @property
-    def available(self):
-        """Could the device be accessed during the last update call."""
-        return self.client.available
+        return self._state.time().strftime(TIME_STR_FORMAT) if self._state else None
 
     @property
     def should_poll(self):
         """Disable polling."""
         return False
 
     @property
-    def device_class(self):
-        """Return the device class."""
-        return DEVICE_CLASS_TIMESTAMP
```

Comment:
Don't remove device class timestamp. In lovelace the user can format how the timestamp should be represented.

https://www.home-assistant.io/lovelace/entities/#format